src/gsAssembler/gsExprAssembler.h

```diff
@@ -651,7 +651,7 @@ gsOptionList gsExprAssembler<T>::defaultOptions()
 }
 
 template<class T>
-void gsExprAssembler<T>::computeDirichletDofs2(int unk)
+void gsExprAssembler<T>::computeDirichletDofs2(index_t unk)
```

Comment:
The variable frequently called "unk" in this file is either int, or short_t, but needs not be index_t. it is usually below 10 and there is sometimes a special value -1.